spec/lib/percy/capybara/loaders/native_loader_spec.rb

```diff
@@ -18,50 +18,50 @@
       loader = described_class.new(page: page)
       resource_urls = loader.snapshot_resources.collect(&:resource_url)
       expect(resource_urls).to match_array([
-        "/test-css.html",
```

Comment:
I'd like to enforce the opposite of this too, to keep it as is. One of my most strong style preferences is to try to never enforce whitespace indentation that has to match lines above, because then if the line length above changes (which it does all the time), I'd have to micromanage the whitespace of the following lines (the same reason why I don't align `= value` lines and always just left-align them).